src/playlist-selectors.js

```diff
@@ -255,44 +255,27 @@ export const movingAverageBandwidthSelector = function(decay) {
  *        Time in seconds left in the forward buffer
  * @param {Number} segmentDuration
  *        Duration of the expected segment
- * @return {Object}
- *         Object with two properties
- *         playlist - The playlist object that will result in the minimum rebuffering
- *         roundTripTime - The estimated amount of time in seconds it will take to
- *         switch to this playlist and download a segment
+ * @return {Array}
+ *         List of playlists with estimated segment round trip times sorted by round trip
+ *         time
  */
 export const minRebufferingSelector = function(master,
```

Comment:
Actually, since it is much smaller, it might be worth inlining it. It would also shorten later code.